pyccel/codegen/printing/ccode.py

```diff
@@ -1639,7 +1639,7 @@ def _print_AliasAssign(self, expr):
             rhs_address = VariableAddress(rhs_var)
         elif isinstance(rhs_var, FunctionCall):
             res = rhs_var.funcdef.results
-            if len(res)!= 0 and res[0].is_pointer:
+            if len(res)!= 0 and res[0].is_alias:
```

Comment:
Did you mean to lose this variable?

Reply:
it should behave the same as `is_pointer`, why it will be lost ?

Reply:
I've just checked. It was my diff